go/vt/sqlparser/sql.y

```diff
@@ -2262,7 +2277,7 @@ rename_statement:
 rename_list:
   table_name TO table_name
   {
-    $$ = &DDL{Action: RenameDDLAction, FromTables: TableNames{$1}, ToTables: TableNames{$3}}
+    $$ = &RenameTable{FromTables: TableNames{$1}, ToTables: TableNames{$3}}
```

Comment:
Instead of pairing from and to tables on index, I think it would be better to create a struct to contain both of them.

Reply:
Done.